Source/Core/Core/AchievementManager.h

```diff
@@ -226,6 +245,10 @@ class AchievementManager
 
 #include <string>
 
+#include "Core/ActionReplay.h"
+#include "Core/GeckoCode.h"
+#include "Core/PatchEngine.h"
+
```

Comment:
These can be removed.

Reply:
Not sure what happened here because these popped back in a couple times but FYI they've been removed and for some reason GitHub isn't marking this as Outdated.

Reply:
Nope, they're still there.

Reply:
Oh my lord, I wasn't paying attention to the line numbers. Visual Studio, why are you like this?